mautrix_instagram/user.py

```diff
@@ -361,7 +362,7 @@ async def logout(self, from_error: bool = False) -> None:
                 pass
             self.igpk = None
         else:
-            await self.push_bridge_state(ok=False, error="logged-out")
+            await self.push_bridge_state(BridgeStateEvent.LOGGED_OUT)
```

Comment:
This one happens for bad credentials, the `if not from_error:` case is when the user logs out.